test/storage/garbage_collector_test.cpp

```diff
@@ -83,7 +83,7 @@ struct GarbageCollectorTests : public ::terrier::TerrierTest {
   storage::BlockStore block_store_{100, 100};
   storage::RecordBufferSegmentPool buffer_pool_{10000, 10000};
   std::default_random_engine generator_;
-  const uint32_t num_iterations_ = 100;
+  const uint32_t num_iterations_ = 1;
```

Comment:
Unintentional change?

Reply:
@wenxuanqiu is correct. Should we set this back to 100?

Reply:
We already set it back to 100. This was committed by mistake.